library/std/src/f16.rs

```diff
@@ -489,6 +491,13 @@ impl f16 {
     /// assert!(abs_difference <= f16::EPSILON);
     /// # }
     /// ```
+    ///
+    /// Non-positive values:
+    /// ```
+    /// #![feature(f16)]
+    /// assert_eq!(0_f16.ln(), f16::NEG_INFINITY);
+    /// assert!((-42_f16).ln().is_nan());
+    /// ```
```

Comment:
Similar to `f128` these need a `reliable_f16_math` gate (see the other examples in this file)